metrics/session.go

```diff
@@ -106,6 +106,7 @@ const (
 	LblOK          = "ok"
 	LblError       = "error"
 	LblRollback    = "rollback"
+	LblComRol      = "com_rol"
```

Comment:
`com_rol` confused some DBA in other releases. I think it is better to do not cherry-pick it.

Reply:
OK